source/code-generation/query-converter.txt

```diff
@@ -0,0 +1,23 @@
+.. _rm-query-converter:
+
+===============
+Query Converter
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use Query Converter to convert SQL statements, views, and stored 
+procedures to MongoDB syntax. 
```

Comment:
I know this is just a stub, but we should emphasise that it isn't a naïve syntax conversion; it actually takes the mapping rules into account.

Reply:
Sure, I added a blurb supporting the fact that the conversions consider your Mapping rules and destination schema.